### DISCUSSION TITLE
Append to current daily note cannot work for the initial creation

### DISCUSSION THREAD

**Discussion by denny0810** [coddingtonbear/obsidian-web#178] [Q&A] [answered]
The function can work well if the current daily note has already existed.  
But when I used it to append somthing to the daily note that was not created yet, it could help to create the daily note file but nothing was appended for the first time. After that, it worked well.
Here is my template for appending to the daily note. Can someone help to find the problem？
```
---
**{{date "HH:mm:ss"}}**
link: *[{{page.title}}]({{page.url}})*
{{#if page.selectedText}}
{{page.selectedText}}
{{/if}}
```

**Comment by coddingtonbear** [comment, marked as answer]
I'm willing to bet that you're not using the ["Periodic Notes" plugin](https://obsidian.md/plugins?id=periodic-notes) and are instead using the default Obsidian "daily note" feature.  If you switch to that plugin instead of the built-in functionality, you should be good-to-go (I use that functionality basically every day).

**Reply by denny0810** [reply]
Hi,
I installed the ["Periodic Notes" plugin](https://obsidian.md/plugins?id=periodic-notes) as suggested and it can work. But I found that in "reading" view, the first captured content can not be displayed.  It started from the second captured contents. 
Then I tried to change the template since I thougth the beginning dash `---` might let the sofware consider the following content would be file properties. 
```
- **{{date "HH:mm:ss"}}**
url: [{{page.title}}]({{page.url}})
{{#if page.selectedText}}
{{page.selectedText}}
{{/if}}
---  
```
It seems the problem is  solved now even I stop the "Periodic Notes" plugin

**Reply by coddingtonbear** [reply]
Obsidian Web isn't involved at all in how "Reading Mode" works in Obsidian, so my first guess would be that there's something wrong in your template.  Unfortunately, I can't make any guesses for you without knowing what your templates look like; so can't help more than I have until you to go to the extension options and upload a bug report export that includes your template presets (that's the "Include a snapshot of your plugin configuration" toggle).
